components/ItemFilter/ItemFilter.tsx

```diff
@@ -399,75 +396,77 @@ function ItemFilter(props: Props) {
+                                                let name = (props.labelKey as Function)(options).toLowerCase()
+                                                let initials = name.match(/\b\w/g).join('')
+                                                let description = options.description ? options.description.replace(/\s/g, '').toLowerCase() : ''
+
+                                                let matchingOptions = options.options.filter(option => option.toLowerCase().includes(searchString))
 
-                                            return `${camelCaseToSentenceCase(name)} ${optionsString ? `(${optionsString})` : ''}`
-                                        }}
-                                        filterBy={(options: FilterOptions, props) => {
-                                            let searchString = props.text.replace(/\s/g, '').toLowerCase()
-                                            let name = (props.labelKey as Function)(options).toLowerCase()
-                                            let initials = name.match(/\b\w/g).join('')
-                                            let description = options.description ? options.description.replace(/\s/g, '').toLowerCase() : ''
-
-                                            let matchingOptions = options.options.filter(option => option.toLowerCase().includes(searchString))
-
-                                            return (
-                                                name.replace(/\s/g, '').includes(searchString) ||
-                                                initials.includes(searchString) ||
-                                                description.includes(searchString) ||
-                                                matchingOptions.length > 0
-                                            )
-                                        }}
-                                        emptyLabel={props.emptyLabel || 'No matches found. Filters which would not show any results are hidden'}
-                                    ></Typeahead>
+                                                return (
+                                                    name.replace(/\s/g, '').includes(searchString) ||
+                                                    initials.includes(searchString) ||
+                                                    description.includes(searchString) ||
+                                                    matchingOptions.length > 0
+                                                )
+                                            }}
+                                            emptyLabel={props.emptyLabel || 'No matches found. Filters which would not show any results are hidden'}
+                                        ></Typeahead>
+                                        <p className={styles.filterHelpText}>You can search for filters both by their name and possible values (if available)</p>
```

Comment:
To keep the UI clean and only let users searching for a specific filter to see this it should be a virtual option in the dropdown itself (selecting it not adding an actual filter)